src/main.rs

```diff
@@ -315,6 +338,62 @@ impl BloomFilter {
     }
 }
 
+
+
+fn compute_bloom_size(fp_rate: f64, expected_ngram_count: usize) -> usize {
+    /* Uses binary search to find optimal size of bloom filter using optimal number of hashers
+       and provided ngram counts
+    */
```

Comment:
That's fine, but is there really no closed solution?

Reply:
Oh maybe, let me do some math real quick

Reply:
Someone smarter than me may solve this
For optimal $k=\frac{m}{n}\log 2$, the relationship between number of items $n$, FP rate $\epsilon$, and filter size $m$ is written as 
$$(1-(1-\frac{1}{m})^{kn})^k = \epsilon$$

Binary search seems easier